docs_user/modules/proc_adopting-compute-services-to-the-data-plane.adoc

```diff
@@ -134,26 +143,11 @@ rm -f id*
 cd -
 ----
 
-. Create a `nova-compute-extra-config` service (with local storage backend for `libvirt`):
-+
-. If TLS Everywhere is enabled, append the following to the OpenStackDataPlaneService spec:
+. If you use a local storage back end for `libvirt`, create a `nova-compute-extra-config` service to remove pre-fast-forward workarounds and configure Compute services to use a local storage back end:
```

Comment:
Is creating a nova-compute-extra-config service with a local storage back end the “default”?
Do we expect customers to create a local storage back end OR a Ceph back end, or both?

Reply:
here, we only focus on two available options (local, and ceph, for Nova's libvirt storage backend) w/o making opinionated choices

Reply:
for nova it is local storage using qcow 2 as it has always been in tripleo and upstream

triloe does not default to ceph either but when ceph was enabled it was enabled for nova,glance and cinder by default unless you overrode that.

for adoption this cant change form whatever the source cloud had

Reply:
Correct, as @SeanMooney mentioned everything depends on the source cloud. In TripleO we had parameters to enable Ceph for Nova ([NovaEnableRbdBackend: true](https://github.com/openstack-archive/tripleo-heat-templates/blob/stable/wallaby/deployment/nova/nova-compute-container-puppet.yaml#L168)), but it's not set to `true` by default and it's role specific.

Reply:
Thank you all for the explanation. I think it it okay to keep the phrasing of the steps, since it depends on their source environment, e.g.,

If you use a local storage back end, <do this>...
If you use a Ceph back end, <do this>...